docs/mantle.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: default
+has_children: true
+has_toc: false
+nav_order: 9
+---
+
+# Mantle
+
+This repository is a collection of utilities for developing Container Linux.
```

Comment:
Can probably do a coarse `s/Container Linux/CoreOS/` throughout these imported docs to start.